### PR TITLE
Adds a testcase for flow_direction

### PR DESCRIPTION
**Pull request by fojac**
The addition of this testcase follows the request on the flow_direction addition (see https://github.com/OISF/suricata/pull/3521).
Hope it is OK for you, as it is my first PR.

### CODE REVIEW

File: tests/flow-direction/test.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+command: |
```
Comment (victorjulien) [inline tests/flow-direction/test.yaml]: with a simple pcap check like this, there is no need to use 'command'.

**Comment by fojac** [comment]
Thanks, so I just have to keep the "checks" part and remove the "command" and "pre-check" lines?
So I understand the suricata.yaml as well as the .pcap should be removed too?

**Comment by catenacyber** [comment]
> So I understand the suricata.yaml as well as the .pcap should be removed too?
The pcap file should be kept.
I think that the suricata.yaml can be removed.

You can add a README.md file to describe the test and explain where the pcap file comes from (or how we can create it from scratch)

File: tests/flow-direction/test.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+pre-check: |
```
Comment (catenacyber) [inline tests/flow-direction/test.yaml]: I think that we do not need this

**Comment by fojac** [comment]
The README.md file was added. The flow direction feature should create the flow direction metadata in nearly any TCP flow. So the PCAP is just there as a test, any usual TCP flow pcap should properly work with this feature, I just added one as requested for the suricata MR of this feature to be accepted.

**Comment by catenacyber** [comment]
> The README.md file was added.

The readme looks clear to me :-)

**Comment by catenacyber** [comment]
@fojac could you rebase this with a minimal yaml file as requested ?

**Comment by catenacyber** [comment]
ping @fojac ?

**Comment by fojac** [comment]
It seems to me the flow_direction feature doesn't work anymore in suricata 7.0 dev.
I've tried to add the patch in output-json.c cloning the latest version of suricata but in eve.json I don't see the flow_direction anymore, so the check fails.

**Comment by catenacyber** [comment]
> doesn't work anymore in suricata 7.0 dev

It does neither with 6.0 or 5.0
Is there a suricata PR related to this ?

**Comment by fojac** [comment]
There was one (https://github.com/OISF/suricata/pull/3521) but now closed.

**Comment by jasonish** [comment]
Closing as it looks like the feature this test is for has gone stale with no recent work. Please re-open if this is not true. Thanks.

**Pull request closed without merging**